lib/puppet_x/puppetlabs/strings/yard/code_objects/method_object.rb

```diff
@@ -0,0 +1,34 @@
+class YARD::CodeObjects::MethodObject
```

Comment:
Would you mind filing a ticket for this improvement so we don't lose track of it?